handlers/common.go

```diff
@@ -89,6 +91,14 @@ func initializeHandlers() *Handler {
 		}
 		time.Sleep(time.Duration(1) * time.Second)
 	}
+	for i := 0; i < 10; i++ {
+		ret, err := utils.GetAgentImage(client)
+		if err == nil {
+			systemImages = ret
+			break
+		}
+		time.Sleep(time.Duration(1) * time.Second)
```

Comment:
if you fail 10 times seems like the program should exit.  Here you just ignore the error after 10 failues